### ISSUE TITLE
How to set up keybindings

### ISSUE THREAD

**Issue by jottr** [koekeishiya/kwm#591]
Maybe it is really obvious and I am just very slow today, but it seems unclear to me how to configure shortcuts/key bindings. 
Is this achieved via [Khd](https://github.com/koekeishiya/khd)?
If so, how?
Wouldn't it make sense to add instructions on how to actually use kwm to the wiki?

**Comment by Amar1729** [comment]
In general, keybindings are handled by `khd` and your `.khdrc`, while the `.kwmrc` is for setting configuration values like default padding. There's a kwmc reference [here](https://koekeishiya.github.io/kwm/kwmc.html).  
Also, if you run `brew info kwm`:  
```
==> Caveats
Copy the example config from /usr/local/opt/kwm/kwmrc into your home directory.
  mkdir -p ~/.kwm
  cp /usr/local/opt/kwm/kwmrc ~/.kwm/kwmrc
```

or `brew info khd`:
```
==> Caveats
Copy the example config from /usr/local/opt/khd/khdrc into your home directory.
  cp /usr/local/opt/khd/khdrc ~/.khdrc
```

The sample config files are fairly informative for creating your own keybindings (I just started with simple modifications to the default ones until I got a better idea of my best workflow).
